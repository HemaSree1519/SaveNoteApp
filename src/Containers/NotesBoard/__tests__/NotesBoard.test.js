import React from 'react';
import { shallow } from 'enzyme';
import Index from '../index';
describe('NotesBoard', () => {
    it('should render correctly', () => {
        const component = shallow(<Index/>);
        expect(component).toMatchSnapshot();
    });
});