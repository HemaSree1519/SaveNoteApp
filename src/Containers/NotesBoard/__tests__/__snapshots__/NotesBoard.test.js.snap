exports[`NotesBoard should render correctly 1`] = `
ShallowWrapper {
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <Index />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "checkPropTypes": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateError": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": Array [
        <div
          className="write-note">
          <Input
                    className="note-input"
                    onClick={[Function]}
                    placeholder="Write a note..."
                    type="text" />
</div>,
        <div>
          <i>
                    Get your note saved with Save Note app
          </i>
</div>,
      ],
      "className": "dash-board",
    },
    "ref": null,
    "rendered": Array [
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": <Input
            className="note-input"
            onClick={[Function]}
            placeholder="Write a note..."
            type="text" />,
          "className": "write-note",
        },
        "ref": null,
        "rendered": Object {
          "instance": null,
          "key": undefined,
          "nodeType": "class",
          "props": Object {
            "className": "note-input",
            "onClick": [Function],
            "placeholder": "Write a note...",
            "type": "text",
          },
          "ref": null,
          "rendered": null,
          "type": [Function],
        },
        "type": "div",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": Array [
            <i>
              Get your note saved with Save Note app
</i>,
            undefined,
          ],
        },
        "ref": null,
        "rendered": Array [
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": "Get your note saved with Save Note app",
            },
            "ref": null,
            "rendered": "Get your note saved with Save Note app",
            "type": "i",
          },
          undefined,
        ],
        "type": "div",
      },
    ],
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <div
            className="write-note">
            <Input
                        className="note-input"
                        onClick={[Function]}
                        placeholder="Write a note..."
                        type="text" />
</div>,
          <div>
            <i>
                        Get your note saved with Save Note app
            </i>
</div>,
        ],
        "className": "dash-board",
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": <Input
              className="note-input"
              onClick={[Function]}
              placeholder="Write a note..."
              type="text" />,
            "className": "write-note",
          },
          "ref": null,
          "rendered": Object {
            "instance": null,
            "key": undefined,
            "nodeType": "class",
            "props": Object {
              "className": "note-input",
              "onClick": [Function],
              "placeholder": "Write a note...",
              "type": "text",
            },
            "ref": null,
            "rendered": null,
            "type": [Function],
          },
          "type": "div",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": Array [
              <i>
                Get your note saved with Save Note app
</i>,
              undefined,
            ],
          },
          "ref": null,
          "rendered": Array [
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": "Get your note saved with Save Note app",
              },
              "ref": null,
              "rendered": "Get your note saved with Save Note app",
              "type": "i",
            },
            undefined,
          ],
          "type": "div",
        },
      ],
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
        "legacyContextMode": "parent",
        "lifecycles": Object {
          "componentDidUpdate": Object {
            "onSetState": true,
          },
          "getChildContext": Object {
            "calledByRenderer": false,
          },
          "getDerivedStateFromError": true,
          "getDerivedStateFromProps": Object {
            "hasShouldComponentUpdateBug": false,
          },
          "getSnapshotBeforeUpdate": true,
          "setState": Object {
            "skipsComponentDidUpdateOnNullish": true,
          },
        },
      },
    },
  },
  Symbol(enzyme.__childContext__): null,
}
`;
